octokit/client.py

```diff
@@ -9,11 +9,13 @@
 
 # https://code.google.com/p/uri-templates/wiki/Implementations
 
-from .resources import Resource
+from resources import Resource
```

Comment:
@tchangg I think what you're doing is running python from the `octokit.py/octokit/` folder instead of the `octokit.py/` folder. The dot before the import tells python to explicitly import from the current package, but it can't do this since the package is supposed to be the `octokit.py/octokit/`folder that you're inside of.

(I was able to reproduce your error by running python from `octokit.py/octokit/`, but it worked fine when running from `octokit.py/`)